mne/beamformer/_dics.py

```diff
@@ -169,7 +170,8 @@ def dics(evoked, forward, noise_csd, data_csd, reg=0.01, label=None,
         If 'normal', rather than pooling the orientations by taking the norm,
         only the radial component is kept.
     verbose : bool, str, int, or None
-        If not None, override default verbose level (see mne.verbose).
+        If not None, override default verbose level (see mne.verbose and
+        :ref:`tut_logging` for more).
```

Comment:
you can format it this way so it stays readable in the console:

:ref:`Logging documentation < tut_logging >`.

use a magic function like this:

```
function git-sedi () {
   git grep -l $1 | xargs sed -i -e "s/$1/$2/g"
}
```
to change it everywhere quickly.